### PR TITLE
docs(calendar): Added sample for calendar views #3855

### PR DESCRIPTION
**Pull request by sboykova**
Related to IgniteUI/igniteui-angular#3855

### CODE REVIEW

File: en/components/calendar.md [hunk 1]
```diff
@@ -323,25 +324,69 @@ Result:
...
+
+When `months` button (in the subheader) is focused, use:
+- <kbd>Space</kbd> or <kbd>Enter</kbd> key to open the months view.
+
+When `year` button (in the subheader) is focused, use:
+- <kbd>Space</kbd> or <kbd>Enter</kbd> key to open the decade view.
+
+When a day inside the current month is focused, use:
+- Arrow keys to navigate through the days,
+- Arrow keys to navigation to previous/next month as well,
```
Comment (SAndreeva) [inline en/components/calendar.md]: May be we should replace **naviagtion** with **navigate**.

File: en/components/calendar.md [hunk 2]
```diff
@@ -54,15 +55,17 @@ Instantiating the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/i
...
     <iframe id="calendar-sample-1-iframe" src='{environment:demosBaseUrl}/scheduling/calendar-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
 <div>
     <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="calendar-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
-###Localization and formatting
+#### Localization and formatting
 
 Due to their very nature, localization and formatting are essential to any calendar. In the [`IgxCalendarComponent`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html) those are controlled and customized through the following properties - [`locale`]({environment:angularApiUrl}/classes/igxcalendarbase.html#locale), [`formatOptions`]({environment:angularApiUrl}/classes/igxcalendarbase.html#formatoptions), [`formatViews`]({environment:angularApiUrl}/classes/igxmonthpickerbase.html#formatviews).
```
Comment (zdrawku) [inline en/components/calendar.md]: Should we change `igxcalendarbase` to `igxcalendarcomponent` here as well

File: en/components/calendar.md [hunk 3]
```diff
@@ -270,8 +272,7 @@ This is the result.
     <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="calendar-sample-6-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
-
-### Special dates
+#### Special dates
 
 [`Special dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#specialdates) feature is using almost the same configuration principles as [`Disabled dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#disableddates). The difference here is dates `styling` and `interaction`. You are able to select and focus [`Special dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#specialdates).
```
Comment (zdrawku) [inline en/components/calendar.md]: Should we change `igxcalendarbase` to `igxcalendarcomponent` here as well

File: kr/components/calendar.md [hunk 4]
```diff
@@ -272,7 +273,7 @@ This is the result.
 </div>
 
 
-### Special dates
+#### Special dates
 
 [`Special dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#specialdates) feature is using almost the same configuration principles as [`Disabled dates`]({environment:angularApiUrl}/classes/igxcalendarcomponent.html#disableddates). The difference here is dates `styling` and `interaction`. You are able to select and focus [`Special dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#specialdates).
```
Comment (zdrawku) [inline kr/components/calendar.md]: Should we change `igxcalendarbase` to `igxcalendarcomponent` here as well

File: en/components/calendar.md (hunk 3 above)
Comment (sboykova) [reply en/components/calendar.md]: igxcalendarbase is the correct one

**Pull request merged**